samples/sdl/Makefile

```diff
@@ -1,6 +1,6 @@
 XBE_TITLE = sdl
 GEN_XISO = $(XBE_TITLE).iso
-SRCS += $(wildcard $(CURDIR)/*.c)
+SRCS += $(CURDIR)/main.c
```

Comment:
This one used to use `+=`, but I think this should just be `=` to be more consistent with other samples.

The httpd sample is also affected by this.